specification/serialconsole/resource-manager/Microsoft.SerialConsole/preview/2018-07-01-preview/examples/GetConsoleDisabledExample.json

```diff
@@ -0,0 +1,12 @@
+{
+    "parameters": {
+        "api-version": "2018-07-01-preview",
+        "subscriptionId": "00000000-00000-0000-0000-000000000000",
+        "default": "default"
+    },
+    "responses": {
+        "200": {
+            "disabled": "true"
```

Comment:
According to the spec, the response should have non empty body. See https://github.com/Azure/azure-rest-api-specs/blob/9a03d2dbc9201acf4956476a1927fcef49b604eb/specification/serialconsole/resource-manager/Microsoft.SerialConsole/preview/2018-07-01-preview/serialconsole.json#L95